packages/block-library/src/media-text/icon-retry.native.js

```diff
@@ -5,7 +5,10 @@ import { Path, SVG } from '@wordpress/components';
 
 export default (
 	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-		<Path d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z" />
+		<Path
+			d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z"
+			fill="#ffffff"
```

Comment:
Hey, I wanted to document what we talked about in our chat a little bit ago.

In image block, we're using a [style](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/image/styles.native.scss#L41-L45) attribute and then setting that on the [icon](https://github.com/WordPress/gutenberg/blob/6e80f44539a03d658935303000aecf313dba9127/packages/block-library/src/image/edit.native.js#L320). 

I think it might be good to continue this pattern instead of setting it in the SVG. Looking at the [media-container](https://github.com/WordPress/gutenberg/blob/6e80f44539a03d658935303000aecf313dba9127/packages/block-library/src/media-text/media-container.native.js#L114) it looks like we're trying to set the [style](https://github.com/WordPress/gutenberg/blob/6e80f44539a03d658935303000aecf313dba9127/packages/block-library/src/media-text/style.native.scss#L70-L74) already. I think it would be worth exploring why that isn't working.

PS. We talked about reusing the retry UI but looking deeper into that suggestion I think that would be an effort for a dedicated issue.

Reply:
Closing PR since it is already handled by https://github.com/WordPress/gutenberg/pull/22788